WBooks/Commons/InputFields/InputFields.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  InputFieldsView.swift
+//  WBooks
+//
+//  Created by sebastian.a.spadea on 12/04/2022.
+//
+
+import Foundation
+import UIKit
+
+@IBDesignable
+open class InputFields: UITextField {
```

Comment:
Access to the class must be conditioned to internal.

Sebas I leave you information about access control, so that you can further evaluate the use.

https://docs.swift.org/swift-book/LanguageGuide/AccessControl.html